actions/actions.py

```diff
@@ -1,15 +1,11 @@
-#!/usr/bin/env python3
+#!/usr/local/sbin/charm-env python3
```

Comment:
TIL; this is the first I've seen /usr/local/sbin/charm-env used, but I see it is [an existing pattern](https://bugs.launchpad.net/charm-tempest/+bug/1760949/comments/1).  +1.